packages/desktop-client/src/components/reports/NetWorth.js

```diff
@@ -133,6 +132,12 @@ export default function NetWorth() {
           start={start}
           end={end}
           graphData={data.graphData}
+          domainMinMax={{
+            y: [
+              data.lowestNetWorth - data.lowestNetWorth * 0.01,
+              data.highestNetWorth + data.highestNetWorth * 0.01,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could be simplified
```suggestion
              data.lowestNetWorth * 0.99,
              data.highestNetWorth * 1.01,
```